### ISSUE TITLE
  Level 777 Media downloaded with wrong size

### ISSUE THREAD

**Issue by adamfernn** [tangyoha/telegram_media_downloader_bak#25]
raise TimeoutError                                      
                    TimeoutError                                                
           Level 777 Media downloaded with wrong size

**Comment by tangyoha** [comment]
@Romntic hi, program can not download success, because it may network not statable.If not do this action, you will lost this media.

**Comment by tangyoha** [comment]
We will add the failed download message id into `ids_to_retry` when have three times `TimeoutError `.

**Comment by adamfernn** [comment]
Yes.I understand but actually if I try with another download method it's success download :(
